client/transactions/filters/config.ts

```diff
@@ -125,12 +126,22 @@ export const getFilters = (
 export const getAdvancedFilters = (
 	customerCurrencyOptions?: TransactionsFilterEntryType[]
 ): any => {
+	// TODO: Remove this and all the checks once we drop support of WooCommerce 7.7 and below.
+	const wooCommerceVersionString = getSetting( 'wcVersion' );
```

Comment:
Will this, in any case, will return `null` or `undefined`?

Reply:
I don't think WCPay can run without WC installed, so I don't think so :) Still, this is a temporary solution that we'll be able to drop once we drop support for 7.7